packages/assets/scss/03-organisms/_sections-three-up.scss

```diff
@@ -107,11 +107,16 @@ $bp-section-3up-large-max: "max-width:950px";
     }
   }
 
-  .ma__vertical-card-group--wrapper {
-
-    .ma__card.ma__card--vertical-card {
-      background-color: $c-white;
+  .ma__card__wrapper {
+    &:hover {
+      border-block-end: 2px solid $c-overcast-gray;
```

Comment:
Please use the semantic colors in colors.scss instead of the primitive color tokens when possible `$c-bd`